src/app/components/PanoramaViewer/PanoramaViewerControls.tsx

```
@@ -17,29 +17,29 @@ type Props = {
   panoImageDate?: string
 }
 
-const ResizeButton = styled(ControlButton)`
+const ResizeButton = styled(Button)`
   @media screen and ${breakpoint('max-width', 'tabletM')} {
     display: none; // below tabletM is always full screen, so no need to show this button
   }
 `
 
 const PanoramaViewerControls: React.FC<Props> = ({ children, onClose, panoImageDate }) => {
-  const { drawerPosition } = useContext(MapPanelContext)
   const { panoFullScreen, setPanoFullScreen } = useContext(MapContext)
   const [location] = useParam(locationParam)
 
+  // TODO: Add the pano viewer to the map overlay.
+
   return (
     <ViewerContainer
-      // @ts-ignore
-      style={!panoFullScreen ? { left: drawerPosition } : {}}
       topRight={
-        <>
-          <ControlButton
+        <Control>
+          <Button
             type="button"
             variant="blank"
             title="Panorama sluiten"
             size={44}
             iconSize={25}
+            // @ts-ignore
```

Why is it necessary to suppress errors here? If we cannot do without error suppression, can you define on this line which errors are suppressed?